packages/esbuild-plugin-html/lib/index.js

```diff
@@ -98,6 +98,25 @@ export default function({ scriptsTarget = 'es2015', modulesTarget = 'es2020' } =
                         await writeFile(outputFile, buffer);
                         outputFile = path.relative(process.cwd(), outputFile);
                         outputFiles.push(outputFile);
+                        addBundleMetafile({
+                            inputs: {
+                                [path.relative(process.cwd(), file)]: {
```

Comment:
We could use the root dir from esbuild options instead of the cwd... not sure if esbuild itself uses root of cwd

Reply:
I tried to reproduce the paths we get in the `else` branch, [here](https://github.com/chialab/rna/pull/17/files#diff-be57db1784d1417716f1ba7b870404158752271da96f3cbb00009828f8058639R142), but couldn't find another way to get the root dir